_data/meltano/extractors/tap-f1/reubenf.yml

```diff
@@ -0,0 +1,32 @@
+capabilities:
+- about
+- batch
+- catalog
+- discover
+- schema-flattening
+- state
+- stream-maps
+description: Highest class of international racing for open-wheel single-seater formula racing cars
+domain_url: https://www.formula1.com/
```

Comment:
Personally, I interpret "domain" as the subject of the tap, which is Formula 1 - not the API that powers the tap (if that makes sense). It is definitely worth mentioning and linking to the Ergast docs, but not sure where best to do that.

Reply:
Yeah usually these two things are the same but since the API is external its a little trickier. I'm fine with leaving it as formual1.com. Maybe adding the Ergast link to the tap readme would help instead.